packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -312,7 +342,7 @@ class FlutterCommandRunner extends CommandRunner<void> {
   List<String> getRepoRoots() {
     final String root = globals.fs.path.absolute(Cache.flutterRoot!);
     // not bin, and not the root
-    return <String>['dev', 'examples', 'packages'].map<String>((String item) {
+    return <String>['dev', 'examples', FlutterGlobalOptions.kPackagesOption].map<String>((String item) {
```

Comment:
I think here the string `'packages'` means a directory name.

Reply:
Oops you're right, I'll change that back